### PR TITLE
add declarations for sharp-timer

### PR DESCRIPTION
**Pull request by afractal**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

**Comment by DanielRosenwasser** [comment]
Any reason you're publishing this to DefinitelyTyped instead of shipping it with your package through npm?

**Comment by afractal** [comment]
Currently I have my types within the npm package, but I want to separate them because I think it`s more appropriate to have my types as a standalone project.

File: types/sharp-timer/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "lib": [
+      "es6"
+    ],
+    "noImplicitAny": false,
```
Comment (DanielRosenwasser) [inline types/sharp-timer/tsconfig.json]: Can you please turn these strict flags (or at least `noImplicitAny`) to `true` and amend the PR?

Comment (afractal) [inline types/sharp-timer/tsconfig.json]: sure

**Comment by DanielRosenwasser** [comment]
Sorry for the delay, thanks!

**Comment by afractal** [comment]
thanks

**Pull request merged**